scene/main/node.cpp

```diff
@@ -1635,6 +1636,12 @@ void Node::remove_child(Node *p_child) {
 	}
 }
 
+void Node::clear_children(bool p_include_internal) {
+	for (Object *n : get_children(p_include_internal)) {
+		((Node *)n)->queue_free();
```

Comment:
Possibly add an option to remove_child as well, to make the children immediately cleared after this call (instead of only after a frame)?